python/electrostatic_force.py

```diff
@@ -0,0 +1,362 @@
+                        text3=canvas.create_text(m,n,text=f"{charge1val}")
+                        line3=canvas.create_line(x2,n,xaxis,yaxis,fill="black",width="2")
+                        f,g=compute.mp(x2,n,xaxis,yaxis)
+                        distnew=int((compute.liner(x2,n,xaxis,yaxis))//60)
+                        distancenew=canvas.create_text(f,g,text=f"{distnew}")
+                        x_11=m
+                        y_11=n                  
+            elif m>730:#right side of the blank bar
+                a1=m-30
+                b1=n-30
+                a2=m+30
+                b2=n+30
+                try:
+                    canvas.delete(distancenew)
+                    canvas.delete(text4)
+                    canvas.delete(oval4)
+                    canvas.delete(line3)
+                    oval4=canvas.create_oval(a1,b1,a2,b2,fill="orange")
+                    text4=canvas.create_text(m,n,text=f"{charge2val}")
+                    line3=canvas.create_line(x_axis,y_axis,a1,n,fill="black",width="2")
+                    f,g=compute.mp(x_axis,y_axis,a1,n)
+                    distnew=int((compute.liner(x_axis,y_axis,a1,n))//60)
+                    distancenew=canvas.create_text(f,g,text=f"{distnew}")
+                    x11=m
+                    y11=n
+                except NameError or UnboundLocalError:    
+                    if flagsmall==0:  
+                        canvas.delete(distance)
+                        canvas.delete(line2)
+                        canvas.delete(oval2)
+                        canvas.delete(text2)
+                        oval4=canvas.create_oval(a1,b1,a2,b2,fill="orange")
+                        text4=canvas.create_text(m,n,text=f"{charge2val}")
+                        line3=canvas.create_line(x_axis,y_axis,a1,n,fill="black",width="2")
+                        x11=m
+                        y11=n  
+                        f,g=compute.mp(x_axis,y_axis,a1,n)
+                        distnew=int((compute.liner(x_axis,y_axis,a1,n))//60)
+                        distancenew=canvas.create_text(f,g,text=f"{distnew}")
+                    elif flagsmall==1:
+                        canvas.delete(line3)
+                        canvas.delete(oval2)
+                        canvas.delete(text2)
+                        canvas.delete(distancenew)
+                        oval4=canvas.create_oval(a1,b1,a2,b2,fill="orange")
+                        text4=canvas.create_text(m,n,text=f"{charge2val}")
+                        line3=canvas.create_line(x_axis,y_axis,a1,n,fill="black",width="2")
+                        x11=m
+                        y11=n
+                        f,g=compute.mp(x_axis,y_axis,a1,n)
+                        distnew=int((compute.liner(x_axis,y_axis,a1,n))//60)
+                        distancenew=canvas.create_text(f,g,text=f"{distnew}")
+        bamchik=i.get()
+        bumchik=mango.get()
```

Comment:
Codacy found an issue: [local variable 'bumchik' is assigned to but never used (F841)](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279653)